### DISCUSSION TITLE
Fingerpring BMW i5 (G30)

### DISCUSSION THREAD

**Discussion by Tase5221** [bimmerconnected/bimmer_connected#707] [Fingerprint data contributions]
[2024-12-11_09-17-48.zip](https://github.com/user-attachments/files/18091902/2024-12-11_09-17-48.zip)

Attached the fingerprints of my car. I hope it helps developing and get myself a few sensors. Since I have only controlls and no sensor ;-)
Best
Clemens

**Comment by rikroe** [comment]
Thanks for the fingerprint! However I'm wondering, you have only controls in Home Assistant and no sensors/selects? From a first peak, all looks good.

Can you please also provide the Home Assistant diagnostics download?
